turbo.json

```diff
@@ -1,14 +1,22 @@
 {
   "$schema": "https://turbo.build/schema.json",
-  "globalDependencies": ["**/.env.*local"],
-  "globalEnv": ["NEXT_PUBLIC_SOCKET_URL", "NEXT_PUBLIC_SUPABASE_URL", "NEXT_SUPABASE_KEY"],
+  "globalDependencies": [
+    "**/.env.*local"
```

Comment:
This is a default turborepo configuration, I just removed the `globalEnv` and when I saved the file the lint adjusted the `globalDependencies` indentation

Reply:
Oh, sure, sorry.

But why remove globalEnv?

Reply:
I thought we wouldn't need it since I was using the package env, but I saw that in the package we received a warning when removing the `globalEnv`. 

so I'll put it back

Reply:
Yes sure, how we handle it on library, is totally useless to repeat it here.

nice, thank you.

Reply:
Lokking in the turboRepo docs, I see this alternative ` "globalDotEnv": [".env"],`
Maybe it will fit better our usecase.

Reply:
https://turbo.build/repo/docs/handbook/environment-variables

Reply:
Yes, it works perfectly